scripts/check-dependency-vulnerabilities.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Display human readable result
+pnpm audit --audit-level high --prod;
+
+# Parse results
+pnpm audit --json --audit-level high --prod > audit-results.json
+high_vul=$(cat audit-results.json | jq '.metadata.vulnerabilities.high')
+critical_vul=$(cat audit-results.json | jq '.metadata.vulnerabilities.critical')
+
+high_vul_threshold=2
+critical_vul_threshold=0
+printf "\nHigh vulnerabilities threshold set at $high_vul_threshold\n"
+echo "Critical vulnerabilities threshold set at $critical_vul_threshold"
+
+if (($high_vul > $high_vul_threshold)) || (($critical_vul > critical_vul_threshold)); then
```

Comment:
PR description mentions "_would fail is set at 2 or greater "High severity" vulnerabilities_". 
From the code, looks like it would only fail for greater than 2. Is that right?

Reply:
Yep, you're right. It's greater than 2. I'll update that in the PR description.